pom.xml

```diff
@@ -171,7 +171,7 @@
         -->
         <jetty-old-kafka.version>9.4.56.v20240826</jetty-old-kafka.version>
         <javax-servlet.version>3.1.0</javax-servlet.version>
-        <strimzi-oauth.version>0.15.0</strimzi-oauth.version>
+        <strimzi-oauth.version>0.16.1</strimzi-oauth.version>
```

Comment:
Shouldn't it fail as 0.16.1 isn't released yet? Also, shouldn't you update the 3rd party libs as well?

Reply:
It's released https://repo1.maven.org/maven2/io/strimzi/kafka-oauth-client/0.16.1/

Reply:
Nope, it is not: https://github.com/strimzi/strimzi-kafka-oauth/releases

Reply:
Ah yes, it wasn't announced yet :-)

Reply:
Anyway, I do not think the announcement is a blocker per se. But the 3rd party libs IMHO are.

Reply:
> But the 3rd party libs IMHO are

Sure I forgot about it. Anyway there is something more I have to add and will explain in the PR soon (so why I labeled as "do-not-merge")